doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Samsung-Artik-5/index.md

```diff
@@ -33,88 +45,89 @@ Install the third-party components as follows.
 
 1. Install the toolchain.
 
+        mkdir artik_root && cd artik_root
+        export ARTIK_ROOT=$(pwd)
         wget https://launchpad.net/linaro-toolchain-binaries/trunk/2013.10/+download/gcc-linaro-arm-linux-gnueabihf-4.8-2013.10_linux.tar.bz2
         tar xvjf gcc-linaro-arm-linux-gnueabihf-4.8-2013.10_linux.tar.bz2
-        sudo mv gcc-linaro-arm-linux-gnueabihf-4.8-2013.10_linux /opt/gcc-linaro-arm-linux-gnueabihf
+        mv gcc-linaro-arm-linux-gnueabihf-4.8-2013.10_linux $ARTIK_ROOT/gcc-linaro-arm-linux-gnueabihf
 
 1. Install zlib (for Avro version higher than 1.7.5).
 
+        cd $ARTIK_ROOT
         wget http://zlib.net/zlib-1.2.8.tar.gz
         tar -xvzf zlib-1.2.8.tar.gz
-        export INSTALLDIR=/opt/gcc-linaro-arm-linux-gnueabihf
+        export INSTALLDIR=$ARTIK_ROOT/gcc-linaro-arm-linux-gnueabihf
```

Comment:
There is a possible issue here. You are exporting this directory into PATH on next line (prepend to the PATH).
After installing of the software into INSTALLDIR/bin, it will be first the first place where _bash_ will look for executables.